src/tests/algorithms_test/calorimetry_CalorimeterClusterRecoCoG.cc

```diff
@@ -97,10 +89,8 @@ TEST_CASE( "the calorimeter CoG algorithm runs", "[CalorimeterClusterRecoCoG]" )
 
 
   for (auto clust : *clust_coll){
-    // require that this cluster's axis is 0,0,1
-    REQUIRE(clust.getShapeParameters()[7] == 0);
-    REQUIRE(clust.getShapeParameters()[8] == 0);
-    REQUIRE(abs(clust.getShapeParameters()[9]) == 1);
+    REQUIRE_THAT(clust.getIntrinsicTheta(), Catch::Matchers::WithinAbs(M_PI * 3 / 4, EPSILON));
+    REQUIRE_THAT(clust.getIntrinsicPhi(), Catch::Matchers::WithinAbs(0, EPSILON));
```

Comment:
Shouldn't this be:
```suggestion
    REQUIRE_THAT(clust.getIntrinsicTheta(), Catch::Matchers::WithinAbs(M_PI / 4, EPSILON));
    REQUIRE_THAT(clust.getIntrinsicPhi(), Catch::Matchers::WithinAbs(M_PI, EPSILON));
```

My mental model is that for a gamma and a perfect detector the cluster intrinsic directions should agree with the direction of the gamma, and electrons it may disagree in phi due to track bending.

This means we need to flip the eigenvector to point away from the origin, or dot product of cluster cog position and principal eigenvector should be positive.

Reply:
You are correct. I was saving the fix for another PR, but can implement here.

Reply:
Descoping here is fine with me. Last week before July, so better to get this in before the rush.

Reply:
Implemented.